e2e-test/cromwell_azure_e2e_test.py

```diff
@@ -66,13 +66,15 @@ def get_workflow_metadata(app_url, workflow_id, bearer_token):
     return perform_get(workflow_endpoint, bearer_token)
 
 # workflow_ids is a deque of workflow ids
-def get_completed_workflow(app_url, workflow_ids, bearer_token, max_retries=4, sleep_timer=60 * 2):
+def get_completed_workflow(app_url, workflow_ids, bearer_token, max_retries=6, sleep_timer=60 * 2):
```

Comment:
> from about 11 minutes to about 15 minutes

Looks more like "from 8 to 12"? Or is there an extra 3 minutes somewhere not accounted for here?

Reply:
We wait five minutes at the beginning before we check at all, then there's an additional `(sleep_timer * max_retries-1)` during which the workflow can finish. `max_retries` is probably more properly called `max_attempts`, since the first check counts as one of the retries. I'll rename it while I'm in there.